tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -41,11 +40,38 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
                 runtime = ProcessInfo.Runtime.NetFx;
             }
 
-            var profilerModule = FindProfilerModule(process);
+            bool isContinuousProfilerEnabled;
 
-            if (profilerModule == null)
+            if (process.EnvironmentVariables.TryGetValue("DD_PROFILING_ENABLED", out var profilingEnabled))
             {
-                Utils.WriteWarning(ProfilerNotLoaded);
+                if (ParseBooleanConfigurationValue(profilingEnabled))
+                {
+                    AnsiConsole.WriteLine(ContinuousProfilerEnabled);
+                    isContinuousProfilerEnabled = true;
+                }
+                else
+                {
+                    AnsiConsole.WriteLine(ContinuousProfilerDisabled);
+                    isContinuousProfilerEnabled = false;
+                }
+            }
+            else
+            {
+                AnsiConsole.WriteLine(ContinuousProfilerNotSet);
+                isContinuousProfilerEnabled = false;
+            }
+
+            var loaderModule = FindLoader(process);
+            var nativeTracerModule = FindNativeTracerModule(process, loaderModule != null);
+
+            if (loaderModule == null)
```

Comment:
If the user is running a pretty old (pre-loader DLL), then the loader isn't relevant and the user could get concerned over this message. What about strengthening the check like this?

```suggestion
            if (loaderModule == null && (nativeTracerModule?.StartsWith("Datadog.Tracer.Native", StringComparison.OrdinalIgnoreCase) ?? true))
```